pkg/cmd/cli/cmd/idle_test.go

```diff
@@ -0,0 +1,316 @@
+package cmd
```

Comment:
Specifically testing the CLI command and its interactions, not necessarily the whole feature (which is what extended tests are for)